native/src/components/MapView.tsx

```diff
@@ -144,6 +158,8 @@ const MapView = ({
     const feature = featureCollection?.features.find((it): it is MapFeature => it.geometry.type === 'Point')
     selectFeature(feature ?? null)
     setSheetSnapPointIndex(1)
+
+    zoomOnClusterPress(pressedLocation)
```

Comment:
You could just pass the first element from `featureCollection` (and even `await mapRef.current.getZoom`) to avoid the checks and the duplicate logic in your new `zoomOnClusterPress` method. Also, isn't this also called for pressing on points, not only clusters?

Reply:
no because you want to center on the cluster and not on the feature in the feature collection thats why i'm using a different layerId (`clusterLayerId`) therefore you get different items
you can just log the `featureCollection` and the `clusterCollection`

```js
Feature undefined
 LOG  ClusterFeature {"geometry": {"coordinates": [10.859384536743164, 48.51899218172588], "type": "Point"}, "id": 138, "properties": {"cluster": true, "cluster_id": 138, "point_count": 2, "point_count_abbreviated": "2"}, "type": "Feature"}

```

Reply:
Thanks, missed that.